hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/OMKeyDeleteResponse.java

```diff
@@ -86,6 +86,9 @@ public void addToDBBatch(OMMetadataManager omMetadataManager,
             isRatisEnabled);
         omMetadataManager.getDeletedTable().putWithBatch(batchOperation,
             ozoneKey, repeatedOmKeyInfo);
+        // Update trashTable in DB.
+        omMetadataManager.getTrashTable().putWithBatch(batchOperation,
```

Comment:
Thank you @bshashikant for the review.
I'm working on this these days, and I would draw a picture for this question.

Reply:
> Ideally, if a key gets deleted from a bucket which is trash enabled, i think we should just add it to the trash table, not the deleted table, otherwise it will lead to having duplicate entries in both trash and delete table. Once the trash interval expires, keys can be moved from trash table to deleted table using a background task which will then be cleaned by the KeyDeleting Service. What do you think?

Yeah, I agree that we shouldn't have duplicated entries in both and  delete table.
This part would be updated .

Reply:
Let's hold off for now on this. There are some ideas being discussed around this.

Reply:
Yeah, thanks @bshashikant for the remind.